src/parse/parse_step_test.go

```diff
@@ -98,12 +98,12 @@ func TestAddDepRescan(t *testing.T) {
 	target1.AddDependency(buildLabel("//package1:target4"))
 
 	// Fake test: calling this now should have no effect because rescan is not true.
-	state.QueueTarget(buildLabel("//package1:target1"), core.OriginalTarget, false, false)
+	state.QueueTarget(buildLabel("//package1:target1"), core.OriginalTarget, false)
 	assertPendingParses(t, state)
 	assertPendingBuilds(t, state) // Note that the earlier call to assertPendingBuilds cleared it.
 
 	// Now running this should activate it
-	rescanDeps(state, map[*core.BuildTarget]struct{}{target1: {}})
+	// rescanDeps(state, map[*core.BuildTarget]struct{}{target1: {}})
 	time.Sleep(time.Millisecond * 100)
```

Comment:
Is any of this needed anymore?

Reply:
Fair point. I've removed the two skipped test cases - there is no plan to bring them back again (I don't think what they are doing is a good enough simulation of reality). TBH I'm a little tempted to drop the whole thing - the rest is a bit similar - but at least they are working for now.